src/Microsoft.OData.Client/DataServiceActionQuery.cs

```diff
@@ -70,6 +71,14 @@ public Task<OperationResponse> ExecuteAsync()
             return Context.ExecuteAsync(this.RequestUri, XmlConstants.HttpMethodPost, Parameters);
         }
 
+        /// <summary>Asynchronously sends the request so that this call does not block processing while waiting for the results from the service.</summary>
+        /// <returns>A task represents the result of the operation. </returns>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        public Task<OperationResponse> ExecuteAsync(CancellationToken cancellationToken)

```

Comment:
Won't this break some existing clients? Due to the public api change?

Reply:
Nope, We added an overloaded method. The original method with no parameters still exist